scheduler/src/cook/mesos/fenzo_utils.clj

```diff
@@ -81,11 +81,11 @@
   (log/debug "Fenzo placement failure information:" failure-results)
   (let [failures-by-job (into {} (map accumulate-placement-failures
                                       failure-results))
-        transactions (mapv (fn [[job-db-id failures-description]]
+        transactions (mapv (fn [[job-uuid failures-description]]
                              (let [failure-summary (str failures-description)]
                                (when (> (count failure-summary) 25000)
                                  (log/warn "Fenzo placement Failure summary is too long!" failure-summary))
-                               {:db/id job-db-id
+                               {:db/id [:job/uuid job-uuid]
```

Comment:
What effect does `[:job/uuid job-uuid]` have in the transaction?

Reply:
Datomic will look up the job with that UUID and replace the ref with that ID.